loader/src/main/java/net/swofty/loader/SkyBlock.java

```diff
@@ -155,7 +155,7 @@ public static void main(String[] args) {
         });
         new Thread(() -> {
             try {
-                Thread.sleep(100);
+                Thread.sleep(3500);
```

Comment:
Not in the main repo, if you want it to be configurable put it in the resources.json but leave it by default 100